app/code/Magento/InventoryApi/Api/Data/StockItemInterface.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Interface StockItemInterface
+ * @api
+ */
+interface StockItemInterface extends ExtensibleDataInterface
```

Comment:
For now we don't have a Service API to retrieve Stock Items **for a given Stock_ID**

That's why for now we can't get StockItems in any case
`
These API should be introduced to InventoryApi\Api\StockItemManagerInterface`